libmscore/cmd.cpp

```diff
@@ -3203,19 +3238,9 @@ void Score::addRemoveBreaks(int interval, bool lock)
 
 void Score::cmdRemoveEmptyTrailingMeasures()
       {
-      MasterScore* score = masterScore();
-      Measure* firstMeasure;
-      Measure* lastMeasure = score->lastMeasure();
-      if (!lastMeasure || !lastMeasure->isFullMeasureRest())
-            return;
-      firstMeasure = lastMeasure;
-      for (firstMeasure = lastMeasure;;) {
-            Measure* m = firstMeasure->prevMeasure();
-            if (!m || !m->isFullMeasureRest())
-                  break;
-            firstMeasure = m;
-            }
-      deleteMeasures(firstMeasure, lastMeasure);
+      auto beginMeasure = firstTrailingMeasure();
```

Comment:
I noticed the cmdRemoveEmptyTrailingMeasures ( Thanks Marc for the pointer ) was a little sloppy here:
already have access to score  in Score:: so don't need masterScore() call, and there's a double statement of firstMeasure = lastMeasure right before doing the same thing in the for loop that doesn't actually use for loop's facilities. Because I needed most of the code to get a "receive first trailing measure" and didn't want to have code duplication, I redid this to clear it up with the new firstTrailingMeasure() function and it interfaces much more cleanly.